avocado/utils/process.py

```diff
@@ -99,10 +100,24 @@ def __str__(self):
             return "CmdError"
 
 
+def normalize_cmd(cmd):
+    """
+    Normalize cmd to be safe for ``shlex.split``
+    """
+    if PY2:
+        if not isinstance(cmd, str):
```

Comment:
Using `six.string_types` is preferred.

Reply:
Not really. I need it to be "str" only as it fails with "unicode". I think it's safer to use `not str` rather than `is str` in case something else not-str will be supplied.

Reply:
OK, I see what you're attempting here, so your point makes sense.